README.md

```diff
@@ -513,21 +513,58 @@ See the `example` cron job included in the `values.yaml` file for a full example
 
 ### Breaking changes
 
-* v6 replaces the zalando postgres operator with cloudnative-pg. This decision was made because cloudnative-pg is meant to be a CNCF project,
-  has increasing popularity and the avoidance of StatefulSets makes management easier. However, there is no direct upgrade path from zalando
-  to cloudnative-pg. Therefore, simply upgrading the helm chart could result in your existing database being deleted and possible data loss.
-
-  Therefore, we recommend first creating a [logical backup](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#logical-backups)
-  of the existing database, and then reimporting that backup to the new database following instructions
+* v6 replaces the [Zalando Postgres
+  operator](https://github.com/zalando/postgres-operator) with
+  [CloudNativePG](https://cloudnative-pg.io/) operator for Postgres. This
+  decision was made because CloudNativePG is a [CNCF](https://www.cncf.io/)
```

Comment:
CloudNativePG is not yet a CNCF project (hence the original wording)

Reply:
Sorry, I read the first paragraph here https://cloudnative-pg.io/blog/running-postgresql-the-kubernetes-way/ and misinterpreted it for being a CNCF project already. Reverted.